### DISCUSSION TITLE
Scalable vector database

### DISCUSSION THREAD

**Discussion by karthiksoman** [BaranziniLab/KG_RAG#13] [Ideas]
Do you have any suggestions for vector databases that can effectively store vectors in the order of tens of millions? With that scale, it should also be capable of real-time retrieval of vectors.
Your suggestions will be appreciated :)

**Comment by billsanto** [comment]
This is a fairly comprehensive spreadsheet of offerings: https://docs.google.com/spreadsheets/d/170HErOyOkLDjQfy3TJ6a3XXXM1rHvw_779Sit-KT7uc/edit#gid=0
 
I asked ChatGPT for options—here is the result: https://chat.openai.com/share/05c6edee-97f1-4fd9-b7e9-9ea0eadf9bca

**Comment by mcvillouta** [comment]
Here is a [table](https://docs.google.com/spreadsheets/d/1BR27oQWXSP0JpX6mqdYz81UaqfoXdQBS8sKZBqNGhZg/edit#gid=0) summarizing the pros and cons of different vector databases and libraries :)

<img width="1079" alt="Screenshot 2023-12-08 at 2 32 58 PM" src="https://github.com/BaranziniLab/KG_RAG/assets/59947961/191ab53f-9e43-4814-89d4-c6fb727f69ab">

**Comment by karthiksoman** [comment]
Thank you @billsanto and @mcvillouta 
The information you both shared is truly valuable for making informed decisions about vector databases.

@mcvillouta it seems you need to provide access to fetch the table that you shared.

**Reply by mcvillouta** [reply]
@karthiksoman, now the link should be public.

**Reply by karthiksoman** [reply]
Thank you @mcvillouta !
